### ISSUE TITLE
Stricter answer checking for init

### ISSUE THREAD

**Issue by kelunik** [composer/composer#5989]
`composer init` currently doesn't enforce `yes|y|no|n` as an answer to `bool` questions like the one below. Somehow I wanted to directly search now multiple times, but that results in a `no` instead of asking again with a hint of valid answers.

**Current**

```
Would you like to define your dependencies (require) interactively [yes]? async-interop
Would you like to define your dev dependencies (require-dev) interactively [yes]? ^C
```

**Expected**

```
Would you like to define your dependencies (require) interactively [yes]? async-interop
Please answer yes, y, no, or n.
Would you like to define your dependencies (require) interactively [yes]?
```

**Comment by tomzx** [comment]
This appears to be fixed @ v1.5.5.

@Seldaek Can you please close this?

**Comment by Seldaek** [comment]
True that, thanks :)
